### ISSUE TITLE
Ability to mark individual mentions as "read"

### ISSUE THREAD

**Issue by tomcardoso** [tomcardoso/sourcerer#391]
Would be great to be able to quickly mark individual alerts in the Mentions view as read. Can currently dismiss them permanently, but not mark as read. Also, the dismiss button is missing some accessibility features (tooltips, keyboard navigation).
